iowrite.cpp

```diff
@@ -1453,12 +1452,14 @@ bool writeGrid(
       phiprof::stop("metadataIO");
       return false;
    }
-
-   //Write Ionosphere Grid
-   if( writeIonosphereGridMetadata( vlsvWriter ) == false ) {
-      phiprof::stop("writeGrid-reduced");
-      phiprof::stop("metadataIO");
-      return false;
+   
+   if( !P::writeFullBGB ) {
```

Comment:
This one might need a little explanatory comment. Why *wouldn't* we also write tthe ionosphere metadata here?

Reply:
(Because we really only use it do write *only* BGB, but that isn't obvious from the code here)

Reply:
I *think* this was so early in now that the ionosphere metadata wasn't actually properly there. Do you want me to test this?

Reply:
Could be tested. And if the ionosphere data isn't there, then let's add a comment along the lines of "If only writing out BGB data, ionosphere metadata isn't valid and should not be output."

Reply:
OK initialising the ionosphere communicator instead to get that written.